src/main/java/com/dell/isg/smi/wsman/command/GetConfigResultsCmd.java

```diff
@@ -50,22 +49,20 @@ public String execute() throws Exception {
 		logger.trace("Entering function: execute()");
 		Addressing doc = session.sendInvokeRequest();
 		Document tempDoc = session.extractAddressBody(doc);
-		String content = XmlHelper.convertDocumenttoString(tempDoc);
-		String output  = content.toString().replaceAll("&gt;", ">");
-		output = output.replaceAll("&lt;", "<");
+		Object content = XmlHelper.findObjectInDocument(tempDoc, "n1:ConfigResults");
 		logger.trace("Exiting function: execute()");
-		return output;
+		return "<Result>" + content + "</Result>";
+	}
+
+	public static void main(String[] args) {
+		try {
+			GetConfigResultsCmd cmd = new GetConfigResultsCmd("100.68.124.31", "root", "calvin", "JID_004752690265");
```

Comment:
Did you intend these values to be hard-coded?  Consider adding an integration test (test ending in IT) instead of the Main method.